### PR TITLE
feat(email): update notification email template to full KCC text

### LINKED ISSUE #442: 🛠 Task: E-mailtekst bijwerken in EmailContentService
# 🛠 Task: E-mailtekst bijwerken in EmailContentService

---

## Intent

De huidige korte, onpersoonlijke notificatietekst vervangen door een heldere, volledige e-mail zodat de KCC-medewerker direct weet wat er van hen verwacht wordt en het bericht past bij de communicatiestijl van het KCC.

---

## Functional Description

De `EmailContentService` genereert momenteel een minimale melding ("Er is een nieuw contactverzoek voor jou. Bekijk contactverzoek {Nummer} in ITA Contactverzoeken."). De HTML-template moet worden bijgewerkt zodat de e-mail de volledige tekst bevat conform de Feature-specificatie: aanhef, toelichting, klikbare link naar het contactverzoek, oproep tot actie en een formele afsluiting.

---

## Design Reference

- Frame: N/A — geen design voor deze task
- Component spec: N/A

---

### In scope

- HTML-template in `EmailContentService` bijwerken naar de volledige e-mailtekst
- Correcte HTML-opmaak met `<p>`-tags voor alinea-scheiding
- Klikbare link met tekst "Contactverzoek [nummer]" die verwijst naar het contactverzoek in ITA

### Out of scope

- Wijzigingen aan SMTP-instellingen of verzendlogica
- Wijzigingen aan de aanroepende services (InternetakenNotifier, ForwardContactRequestService)
- E-mail subject-regel aanpassen

---

### Request

```csharp
// Geen wijziging aan de interface
string BuildInternetakenEmailContent(Internetaak internetaak, string itaBaseUrl);
```

### Response

```html
<html>
  <head>...</head>
  <body>
    <p>Beste collega,</p>
    <p>Er is een vraag of verzoek binnengekomen van een inwoner of ondernemer.<br/>
    Je kunt het verzoek bekijken en afhandelen via onderstaande link:</p>
    <p><a href="{deeplink}">Contactverzoek {Nummer}</a></p>
    <p>We verzoeken je om dit zo spoedig mogelijk op te pakken.</p>
    <p>Met vriendelijke groet,<br/>KCC/Klantcontactcentrum</p>
  </body>
</html>
```

### Events / Side Effects

Geen wijziging — de e-mail wordt verstuurd via het bestaande SMTP-pad.

---

## Technical Approach

- Architecture decisions: pure content-wijziging in de bestaande `EmailTemplate` const-string binnen `EmailContentService.cs`
- Data flow: `Internetaak.AanleidinggevendKlantcontact.Nummer` + `itaBaseUrl` → HTML-string via `StringBuilder.Replace()` (ongewijzigd)
- Trade-offs: `<p>`-tags voor alinea-scheiding i.p.v. `<br/>` — betere rendering in diverse e-mailclients

### Aggregates & Invariants

- Aggregates: geen — dit is een read-only service die content genereert
- Invariants: het AanleidinggevendKlantcontact.Nummer moet aanwezig zijn (bestaande validatie blijft ongewijzigd)

---

## Impacted Areas

- Backend: `InterneTaakAfhandeling.Common/Services/Emailservices/Content/EmailContentService.cs`
- Frontend: N/A
- Database: N/A
- APIs: N/A
- External systems: N/A

---

## Gherkin

```gherkin
Feature: Heldere notificatie-e-mail bij toewijzing contactverzoek

  Scenario: Notificatie-e-mail bevat volledige tekst conform specificatie
    Given een contactverzoek is toegewezen aan een KCC-medewerker
    And het AanleidinggevendKlantcontact heeft Nummer "KLC-2024-001"
    When de notificatie-e-mail wordt opgesteld
    Then bevat de e-mail de aanhef "Beste collega,"
    And bevat de e-mail de hoofdtekst "Er is een vraag of verzoek binnengekomen van een inwoner of ondernemer."
    And bevat de e-mail de instructie "Je kunt het verzoek bekijken en afhandelen via onderstaande link:"
    And bevat de e-mail een klikbare link met tekst "Contactverzoek KLC-2024-001"
    And bevat de e-mail het verzoek "We verzoeken je om dit zo spoedig mogelijk op te pakken."
    And bevat de e-mail de afsluiting "Met vriendelijke groet," gevolgd door "KCC/Klantcontactcentrum"

  Scenario: HTML-opmaak scheidt alinea's correct met witruimte
    Given een contactverzoek is toegewezen aan een KCC-medewerker
    When de notificatie-e-mail wordt opgesteld
    Then worden de aanhef, hoofdtekst, link, verzoek en afsluiting als afzonderlijke alinea's weergegeven
```

---

## Edge Cases & Risks

- **HTML-opmaak bij regeleinden** — de tekst bevat meerdere alinea's; elke sectie (aanhef, body, link, verzoek, afsluiting) moet als eigen `<p>`-blok worden weergegeven voor correcte rendering in e-mailclients

---

## Observability

- Logs: bestaande logging in `InternetakenNotifier` en `ForwardContactRequestService` volstaat
- Metrics: geen nieuwe metrics nodig
- Alerts: geen nieuwe alerts nodig

---

## Dependencies

Geen dependencies — deze task is onafhankelijk implementeerbaar.

---

## Rollout

- Feature flag: N/A
- Backward compatibility: N/A — de wijziging is puur cosmetisch in de e-mailinhoud
- Data migration: N/A

---

## Definition of Done

- [ ] Contracts match implementation
- [ ] All Gherkin scenarios covered by tests
- [ ] Edge cases handled
- [ ] Observability in place
- [ ] Code reviewed
- [ ] Design reviewed (if UI)
- [ ] Documentation updated

---

## Test Mapping

| Gherkin Scenario | Test file | Status |
| ---------------- | --------- | ------ |
| Notificatie-e-mail bevat volledige tekst | E2E (Phase 2) | ⏳ pending deploy |
| HTML-opmaak scheidt alinea's correct | E2E (Phase 2) | ⏳ pending deploy |

### PR DESCRIPTION
**Pull request by nijmra**
## Summary

De huidige onpersoonlijke notificatietekst ("Er is een nieuw contactverzoek voor jou") is vervangen door de volledige KCC-communicatie inclusief aanhef, toelichting, klikbare link, oproep tot actie en formele afsluiting. Dit zorgt ervoor dat de KCC-medewerker direct weet wat er verwacht wordt en het bericht past bij de communicatiestijl van het KCC.

## Changes

- **E-mail template bijgewerkt** — HTML-template in `EmailContentService` uitgebreid met volledige tekst: "Beste collega," aanhef, body met uitleg, link naar contactverzoek (alleen nummer klikbaar), verzoek tot actie, en "Met vriendelijke groet, KCC/Klantcontactcentrum" afsluiting
- **HTML-structuur verbeterd** — `<p>`-tags voor correcte alinea-scheiding in e-mailclients

## Test plan

- [ ] Notificatie-e-mail bevat volledige tekst conform specificatie
- [ ] HTML-opmaak scheidt alinea's correct met witruimte

## Related

Closes #442

### CODE REVIEW

File: InterneTaakAfhandeling.Common/Services/Emailservices/Content/EmailContentService.cs [hunk 1]
```diff
@@ -21,8 +21,12 @@ public class EmailContentService : IEmailContentService
         </style>
     </head>
     <body>
-        <p>Er is een nieuw contactverzoek voor jou.</p>
-        <p>Bekijk <a href=""{Link}"">contactverzoek {Nummer} in ITA Contactverzoeken</a>.</p>
+        <p>Beste collega,</p>
+        <p>Er is een vraag of verzoek binnengekomen van een inwoner of ondernemer.<br/>
+        Je kunt het verzoek bekijken en afhandelen via onderstaande link:</p>
```
[the remark this answers is not in the document]
Comment (nijmra) [reply InterneTaakAfhandeling.Common/Services/Emailservices/Content/EmailContentService.cs]: Fixed: 9422f02
